mods/Fifty.ServerTeamBalance/mod/scripts/vscripts/fstb_command_callbacks.nut

```diff
@@ -12,33 +12,35 @@ void function FSTM_CommandCallback_Switch( entity player, array< string > args )
 		return
 	}
 
-	entity target = player
-
+	entity target
 	if( FSA_IsAdmin( player ) && args.len() != 0 ) {
-		bool foundTarget = false
 		foreach( entity p in GetPlayerArray() ) {
 			if( p.GetPlayerName().tolower().find( args[0].tolower() ) != -1 ) {
+				if( target != null ){
+					FSU_PrivateChatMessage(player, "%EMore than one matching player! %TWrite a bit more of their name.")
+					return
+				}
 				target = p
-				foundTarget = true
-				break
 			}
 		}
-
-		if( !foundTarget ) {
+		if(target == null){
 			FSU_PrivateChatMessage( player, "%H\"" + args[0] + "\"%E couldn't be found!" )
 			return
 		}
+		else
```

Comment:
you used braces for the `if`, use them for the `else` too even if it's just one line